recipes-bsp/u-boot/files/0011-board-siemes-iot2050-use-the-named-gpio-to-control-t.patch

```diff
@@ -0,0 +1,30 @@
+From 41a9e985931f425c2a6b25d529a08114dddddd85 Mon Sep 17 00:00:00 2001
+From: chao zeng <chao.zeng@siemens.com>
+Date: Wed, 12 Jan 2022 10:34:18 +0800
+Subject: [PATCH 11/14] board:siemes:iot2050: use the named gpio to control the
+ user-button
+
+Has already add the gpio controll base as the prefix, using the named gpio
+to control the user button
```

Comment:
Here, the benefit of this change should be explained.

Reply:
> Here, the benefit of this change should be explained.
there are three zone, each gpio number starts at 0. 
at here,just using an optical number to control the gpio does not accurately identify a specific pin. like 25, maybe wkup main0 main1? we don't know. Use the prefix to identify which one is used